src/components/ReceiveTableContainer/ReceiveTable/TableUpdater.tsx

```diff
@@ -0,0 +1,74 @@
+import { getPacket, isNumericMeasurement } from "common";
+import { useInterval } from "hooks/useInterval";
+import { createContext, useRef } from "react";
+import { store } from "store";
+
+export type PacketElement = {
+    count: Text;
+    cycleTime: Text;
+};
+
+type MeasurementElement = { value: Text };
+
+type Updater = {
+    addPacket: (id: number, element: PacketElement) => void;
+    removePacket: (id: number) => void;
+    addMeasurement: (id: string, element: MeasurementElement) => void;
+    removeMeasurement: (id: string) => void;
+};
+
+export const TableContext = createContext<Updater>({
+    addPacket() {},
+    removePacket() {},
+    addMeasurement() {},
+    removeMeasurement() {},
+});
+
+type Props = {
+    children?: React.ReactNode;
+};
+export const TableUpdater = ({ children }: Props) => {
+    const packetElements = useRef<Record<string, PacketElement>>({});
+    const measurementElements = useRef<Record<string, MeasurementElement>>({});
+
+    useInterval(() => {
+        const state = store.getState();
+        const podData = state.podData;
+
+        for (const id in packetElements.current) {
+            const packet = getPacket(podData, Number.parseInt(id));
+            const element = packetElements.current[id];
+            element.count.nodeValue = packet.count.toFixed(0);
+            element.cycleTime.nodeValue = packet.cycleTime.toFixed(0);
+        }
+
+        for (const id in measurementElements.current) {
+            const measurement = state.measurements[id];
+            measurementElements.current[id].value.nodeValue =
+                isNumericMeasurement(measurement)
+                    ? measurement.value.average.toFixed(3)
+                    : measurement.value.toString();
+        }
+    }, 1000 / 60);
```

Comment:
Magic number. It is better to define this outside as a constant with a name that explains what that value is (const TICK_RATE = 1000/60 or whatever)